integration/cypress/integration/login/login.spec.ts

```diff
@@ -11,7 +11,7 @@ context("Login page", () => {
     clearCookies();
   });
 
-  it("has no detectable accessibility violations on load", () => {
+  it.skip("has no detectable accessibility violations on load", () => {
```

Comment:
Can you plz add a why to this? 

`// XXX This is skipped because.....`

Reply:
Pinned this down to a missing html lang attribute (only on CI run), more info in the issue https://github.com/canonical-web-and-design/maas-ui/issues/3305